docs/zh/manuals/atlas.md

```diff
@@ -114,3 +114,92 @@ Playback
   - `Loop Forward` 从第一张图片到最后一张图片循环播放.
   - `Loop Backward` 从最后一张图片到第一张图片循环播放.
   - `Loop Ping Pong` 从第一张图片播放到最后一张图片再反向循环播放.
+
+## 运行时纹理及图集建立
+
+自从 Defold 1.4.2 版本, 可以在运行时创建纹理和图集.
+
+### 在运行时创建纹理资源
+
+使用 (`resource.create_texture(path, params)`)[https://defold.com/ref/stable/resource/#resource.create_texture:path-table] 创建纹理资源:
```

Comment:
I used the wrong markdown for this link. It should be []() and not ()[]. Could you please change it?

Reply:
ok, will fix it latter.